style.css

```diff
@@ -166,11 +184,40 @@ footer{
         font-size: 65%;
     }
     .carousel{
-        min-height: 25vh;
+        min-height: 35vh;
         position:relative;
     }
     .hamburger-wrapper{
         display:block;
-        margin:1.5rem 1rem;
+        margin: auto .3rem;
+        font-size: 1.5rem;
+    }
+    .page-link-wrapper{
+        display:none;
+    }
+    #user-icon{
+        display:none;
+    }
+    .side-section{
+        display:block;
+        position:fixed;
+        top:0;
+        left:-50%;
+        background-color: var(--bg-off-white);
+        z-index: 3;
+        width:50%;
+        min-height:100vh;
+        font-size:2rem;
+        transition: all 500ms linear;
+    }
+    .navbar-items>li{
```

Comment:
Looking good. Also add cursor pointer to list item and cancel button.

Reply:
Done! thanks for reviewing :)